_maps/map_files/Delta/delta.dmm

```diff
@@ -85373,7 +85426,7 @@
 /obj/machinery/door/airlock/mining/glass{
 	name = "Supply Break Room"
 	},
-/obj/effect/mapping_helpers/airlock/access/all/supply/general,
+/obj/effect/mapping_helpers/airlock/access/all/supply/mail_sorting,
```

Comment:
These shouldn't be changed to mail_sorting, that's meant for the Delivery Office only

Reply:
so you want me just remove the airlock_helper on that door instead?

Reply:
fixed